tests/Microsoft.Identity.Client.Extensions.Msal.UnitTests/MacKeyChainTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Identity.Client.Extensions.Msal.UnitTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Xunit.Sdk;
+
+namespace Microsoft.Identity.Client.Extensions.Msal
+{
+    [TestClass]
+    public class MacKeyChainTests
```

Comment:
assuming these tests doesn't really run today as we don't run the utests on mac. Also assuming that Mac stuff is not "released" - perhaps at least a comment should be added about the mac state?

Reply:
Sure.  Yes, the mac tests don't run on our environment, but they can be run if you have a mac.  The other tests pass.  I'll add a comment about the status of the mac (and linux) code from a testing and functionality standpoint.